profiles/common/modules/custom/multisite_cookie_consent/multisite_cookie_consent.module

```diff
@@ -43,35 +43,47 @@ function multisite_cookie_consent_menu() {
 }
 
 /**
- * Implements hook_init().
+ * Implements hook_page_build().
  */
-function multisite_cookie_consent_init() {
+function multisite_cookie_consent_page_build(&$page) {
   // Don't display banner in cookie page.
   if (current_path() != 'cookie-notice') {
     // Notice url.
     $notice_page = url('cookie-notice', array('absolute' => TRUE));
     // Add for example external.
-    drupal_add_js('http://ec.europa.eu/wel/like/js/likedislike.js', 'external');
-    // Add js inline (example for now).
-    drupal_add_js('var consent_required_cookies = {
+    $page['content']['#attached']['js'][] = array(
+      'type' => 'external',
+      'data' => 'http://ec.europa.eu/wel/like/js/likedislike.js',
```

Comment:
likedislike.js is deprecated and removed on release-2.5
We can change the like by http://europa.eu/webtools/load.js

Reply:
Hi @voidtek,
I'm not keen on changing anything than strictly necesary in this module since it is foreseen to remove it anyways.